public/src/overmind/utils/errors.ts

```diff
@@ -0,0 +1,30 @@
+import { Code } from "@bufbuild/connect"
+import { AnyMessage } from "@bufbuild/protobuf"
+import { Response } from "../../client"
+
+/** Prompt contains the messages to display to the user when prompting for confirmation. */
+export namespace Prompt {
+    export const GroupDelete = "Are you sure you want to delete this group?"
+    export const GroupRepoNotEmpty = "Warning: The group repository is not empty. Do you still want to delete the group, GitHub team, and group repository?"
+    export const EnrollmentRepoNotEmpty = "Warning: The enrollment repository is not empty. Do you still want to delete the enrollment and enrollment repository?"
```

Comment:
Not sure if `enrollment` is a "user-facing" term. Does this message apply to both group and user repos? If only user, we could just go with that.